### ISSUE TITLE
ValueError: invalid literal for int() with base 10: '0b2'

### ISSUE THREAD

**Issue by mihow** [kvesteri/sqlalchemy-utils#649]
When trying to use sqlalchemy-utils with SqlAlchemy 2.0b2 the version check seems to fail

```
    sqlalchemy_version = tuple([int(v) for v in __version__.split(".")])
  File "lib/python3.9/site-packages/sqlalchemy_utils/types/uuid.py", line 8, in <listcomp>
    sqlalchemy_version = tuple([int(v) for v in __version__.split(".")])
ValueError: invalid literal for int() with base 10: '0b2'
```

**Comment by kurtmckee** [comment]
Thanks for reporting this, @mihow! This should be fixed by #644.

@kvesteri It might be worthwhile to push a release with this bug fix.

**Comment by mihow** [comment]
Thanks for the quick work! I look forward to the release. Right now I am monkey patching sqlalchemy to report the version as "2.0.0" :)
